tslint.json

```diff
@@ -1,14 +1,21 @@
 {
   "rulesDirectory": [
-    "node_modules/codelyzer"
+    "codelyzer"
   ],
   "rules": {
     "member-access": true,
     "member-ordering": [
       true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+      {
```

Comment:
This seems consistent with https://palantir.github.io/tslint/rules/member-ordering/, is this an updated syntax to what we currently have?